lib/puma/control_cli.rb

```diff
@@ -33,10 +33,10 @@ class ControlCLI
     COMMANDS = CMD_PATH_SIG_MAP.keys.freeze
 
     # commands that cannot be used in a request
-    NO_REQ_COMMANDS = %w{refork}.freeze
+    NO_REQ_COMMANDS = %w[].freeze
```

Comment:
Probably should just remove this constant then, yes?

Reply:
I feel like I'm missing something.

Reply:
I can, but I thought I'd leave the code in case a 'signal only' command was added.  Or, previous to this PR, `refork` was a signal only command, and I never tested it using a control server that had a control uri.

Leaving as is means that CI will check that if one is added.  Ok, I know it's a little odd...

Reply:
I commented out the constant and the clause that used it.  Same for the test (skipped if the constant is undefined).

I can remove if you want...  I just don't want myself or someone else making the same mistake again, so the commented code leave a **big** hint...